### DISCUSSION TITLE
An empty workspace

### DISCUSSION THREAD

**Discussion by mcnuggetsx20** [qtile/qtile#4014] [Q&A]
Is there  a way to make an empty workspace? I mean a workspace without bars etc. It would be cool if I were also able to set a different wallpaper for that one workspace but just a black screen would also be fine.

**Comment by elParaguayo** [comment]
You could use a function that's hooked to a group change. When a particular group is opened you could then hide the bar and change the wallpaper. Changing to any other group could reset those options in the same function.

**Comment by mcnuggetsx20** [comment]
Sounds good but how do I detect which group I'm switching to?

**Reply by elParaguayo** [reply]
Actually, thinking about this some more, you'd need to write a function to replace the one that switches group so that it makes the changes above and then switches group. You would then bind this to your keys so that the function receives the name of the target group.

**Reply by mcnuggetsx20** [reply]
It's almost done but how do I change groups without the `lazy` object? All I have inside a function is the 'qtile' object.

**Reply by elParaguayo** [reply]
If you're on v0.22.1 it's `qtile.groups_map[groupname].cmd_toscreen()`. If you're on git latest it's `qtile.groups_map[groupname].toscreen()`.

`groupname` is a string.
